src/test/ui/issues/issue-60662.rs

```diff
@@ -0,0 +1,9 @@
+// compile-flags: -Z unpretty=hir
```

Comment:
Feature gate is missing (`#![feature(existential_type)]`)

Reply:
Added that, thanks!